### ISSUE TITLE
How is model ranking measured?

### ISSUE THREAD

**Issue by kieran199** [automl/auto-sklearn#1207]
Hi there

Is the model accuracy for a classifier to determine rank actually accuracy, or is it based on F1?

Is it possible to adjust it to seek out the best F1 score for a specific class? Or the best average F1 scroes across classes?

This is an amazing library by the way!

THanks

**Comment by eddiebergman** [comment]
The rank is dependent upon the `cost` column which is what is calculated from the metric used.
The defaults depends on the task, multi-label classification is F1 while binary classification is accuracy.
For example, an accuracy of .7 incurs a `cost` of .3.

You can pass your [own metric](https://automl.github.io/auto-sklearn/master/api.html#autosklearn.classification.AutoSklearnClassifier) during construction to do as you need as seen in [this example](https://automl.github.io/auto-sklearn/master/examples/40_advanced/example_calc_multiple_metrics.html#metrics)
